tgui/packages/tgui/interfaces/CrewMonitor.js

```diff
@@ -200,7 +200,7 @@ const CrewMonitorMapView = (_properties, context) => {
   const { act, data } = useBackend(context);
   const { highlightedNames } = data;
   return (
-    <Box height="526px" mb="0.5rem" overflow="hidden">
+    <Box height="5260px" mb="0.5rem" overflow="hidden">
```

Comment:
I dont like that this is just an arbitrarily large number. I think an alternate solution like turning this into a flex component or surrounding the box with a `<Section fill><Box...></ Section>` is required.

Reply:
Your suggestion worked perfectly with height="100vh"!